docs/developer/app-store/apps/adyen/overview.mdx

```diff
@@ -266,3 +266,35 @@ mutation TransactionProcess($transactionId: ID!, $data: JSON, $customerIpAddress
   }
 }
 ```
+
+## Data processing
+
+The app *processes* data that may be PIIs or sensitive:
+
+### Processing
+
+App is retrieving and sending data to Adyen with modification or persisting:
+
+- The IP address of the customer
+- Browser data that Adyen drop-in is gathering (app is receiving it from the Storefront and sending it to Adyen). See [Adyen docs](https://docs.adyen.com/online-payments/analytics-and-data-tracking/)
+- Saleor customer ID
+- Checkout / order language
+- Line items from checkout / order (SKUs, descriptions, etc)
+- Customer email
+- Customer name, surname
+- Customer phone
+- Customer billing and shipping address
+
```

Comment:
Suggestion:

The app does not process any Cardholder Data (CHD) or Sensitive Authentication Data (SAD), as defined by the PCI DSS regulations. Such information is exchanged directly between the Adyen storefront components and the Adyen servers.